src/pages/gas.tsx

```diff
@@ -0,0 +1,474 @@
+                  a transaction.
+                </ListItem>
+              </UnorderedList>
+              <Text>
+                The formula for calculating a gas fee is units of gas used *
+                (base fee + priority fee). Most wallets will calculate gas usage
+                and display it in a more straight-forward way.
+              </Text>
+            </Box>
+            <Table maxW={"100%"} minW={"auto"}>
+              <Thead>
+                <Tr>
+                  <Th>Transaction type</Th>
+                  <Th>Gas used</Th>
+                </Tr>
+              </Thead>
+              <Tbody>
+                <Tr>
+                  <Td>Simple ETH transaction</Td>
+                  <Td>21,000</Td>
+                </Tr>
+                <Tr>
+                  <Td>Transferring an NFT</Td>
+                  <Td>84,904</Td>
+                </Tr>
+                <Tr>
+                  <Td>Swapping on Uniswap</Td>
+                  <Td>184,523</Td>
+                </Tr>
+              </Tbody>
+            </Table>
+          </Flex>
+        </Content>
+
+        {/* Faq section */}
+        <Content>
+          <H2 mt="0">Frequently asked questions</H2>
+          {/* MaxWidth will be enforced by FAQ component once implemented */}
+          <Box maxWidth="832px">
+            <ExpandableCard title="Who gets paid the gas fee in my transaction?">
+              <Text>
+                The majority is gas fee—the base fee— is destroyed by the
+                protocol (burned). The priority fee, if included in your
+                transaction, will be given to the validator who proposed your
+                transaction.
+              </Text>
+              <Text>
+                You can read a detailed description of the process in{" "}
+                <Link href="/developers/docs/gas/">
+                  the gas developer docs.
+                </Link>
+              </Text>
+            </ExpandableCard>
+            <ExpandableCard title="Do I need to pay gas in ETH?">
+              <Text>
+                Yes. All gas fees on Ethereum must be paid in the native ETH
+                currency.
+              </Text>
+              <Link href="/eth/">More on ETH</Link>
+            </ExpandableCard>
+            <ExpandableCard title="What is gwei?">
+              <Text>
+                In some wallets or gas trackers, you will see gas prices
+                denominated as ‘gwei’.
+              </Text>
+              <Text>
+                Gwei is just a smaller unit of ETH, just as pennies are to
+                dollars, with the difference being that 1 ETH equals 1 billion
+                gwei. Gwei is useful when talking about very small amounts of
+                ETH.
+              </Text>
+            </ExpandableCard>
+          </Box>
+        </Content>
+      </>
+      <Divider mb={0} />
+      <Box mt={0}>
+        <QuizWidget quizKey="layer-2" />
```

Comment:
Unfortunately, we can't have a quiz until we fix the bugs in the quiz hub: 

https://github.com/ethereum/ethereum-org-website/pull/10873#issuecomment-1667492697

Reply:
Quiz Questions: 

```json
"i001-prompt": "What causes high gas fees? (easy)",
"i001-a-label": "Ethereum developers intentionally increase them",
"i001-a-explanation": "Ethereum developers do not intentionally increase gas fees. The fees are determined by network demand.",
"i001-b-label": "The price of ETH",
"i001-b-explanation": "The price of ETH is not directly responsible for high gas fees. While the value of ETH can influence the overall cost of a transaction, a high ETH price doesn't necessarily equate to high gas fees, and vice versa.",
"i001-c-label": "High network demand and congestion",
"i001-c-explanation": "High gas fees on Ethereum are caused by high demand for block space and network congestion.",
"i001-d-label": "The price of gasoline",
"i001-d-explanation": "This is a misunderstanding. Ethereum's 'gas' is a unit for measuring computational work, not a physical fuel.",

"i002-prompt": "Layer 2s:",
"i002-a-label": "Have typically lower fees than Ethereum",
"i002-a-explanation": "Layer 2s do have typically lower fees than Ethereum, but this is not the only correct answer.",
"i002-b-label": "Help to scale Ethereum",
"i002-b-explanation": "Layer 2s do help Ethereum scale, but this is not the only correct answer.",
"i002-c-label": "Allow Ethereum to handle more transactions",
"i002-c-explanation": "Layer 2s do allow Ethereum to handle more transactions, but this is not the only correct answer.",
"i002-d-label": "All of the above",
"i002-d-explanation": "Layer 2s are scaling Ethereum, allowing it to handle more transactions with fees usually much lower than Ethereum Mainnet.",

"i003-prompt": "Network fees are also known as:",
"i003-a-label": "Gas fees",
"i003-a-explanation": "Network fees on Ethereum are typically called gas fees.",
"i003-b-label": "Tokens",
"i003-b-explanation": "Tokens are not directly relevant to network fees.",
"i003-c-label": "Gas",
"i003-c-explanation": "Gas is a measure of computational action. A combination of all gas used in a transaction (gas fees) is the network fee.",
"i003-d-label": "NFTs",
"i003-d-explanation": "NFT is a type of non-fungible token on Ethereum. It is not directly relevant to network fees.",

"i004-prompt": "How much gas does sending ETH use?",
"i004-a-label": "1000",
"i004-a-explanation": "Sending ETH is one of the least costly gas operations on Ethereum, but it costs more than 1000 gas.",
"i004-b-label": "21,000",
"i004-b-explanation": "Sending ETH costs exactly 21,000 gas!",
"i004-c-label": "84,904",
"i004-c-explanation": "84,904 is the typical cost for an NFT (ERC-721). Sending ETH costs less gas than this.",
"i004-d-label": "1,088,000",
"i004-d-explanation": "1,088,000 is the typical cost for setting up a RocketPool validator. Sending ETH costs less gas than this.",

"i005-prompt": "How much gas does a smart contract use?",
"i005-a-label": "It depends on the price of ETH",
"i005-a-explanation": "The price of ETH has no correlation with how much gas is used.",
"i005-b-label": "All smart contracts consume exactly 84,904 gas.",
"i005-b-explanation": "Smart contracts do not consume a fixed amount of gas.",
"i005-c-label": "Smart contracts consume different amounts of gas depending on their complexity.",
"i005-c-explanation": "Correct! A complex smart contract with many operations will consume more gas than a simple one.",
"i005-d-label": "All of the above",
"i005-d-explanation": "Only one of the answers is correct."  
```

Question Bank:

```
// Gas
"i001": {
  "prompt": "i001-prompt",
  "answers": [
    {
      "id": "i001-a",
      "label": "i001-a-label",
      "explanation": "i001-a-explanation"
    },
    {
      "id": "i001-b",
      "label": "i001-b-label",
      "explanation": "i001-b-explanation"
    },
    {
      "id": "i001-c",
      "label": "i001-c-label",
      "explanation": "i001-c-explanation"
    },
    {
      "id": "i001-d",
      "label": "i001-d-label",
      "explanation": "i001-d-explanation"
    }
  ],
  "correctAnswerId": "i001-c"
},
"i002": {
  "prompt": "i002-prompt",
  "answers": [
    {
      "id": "i002-a",
      "label": "i002-a-label",
      "explanation": "i002-a-explanation"
    },
    {
      "id": "i002-b",
      "label": "i002-b-label",
      "explanation": "i002-b-explanation"
    },
    {
      "id": "i002-c",
      "label": "i002-c-label",
      "explanation": "i002-c-explanation"
    },
    {
      "id": "i002-d",
      "label": "i002-d-label",
      "explanation": "i002-d-explanation"
    }
  ],
  "correctAnswerId": "i002-d"
},
"i003": {
  "prompt": "i003-prompt",
  "answers": [
    {
      "id": "i003-a",
      "label": "i003-a-label",
      "explanation": "i003-a-explanation"
    },
    {
      "id": "i003-b",
      "label": "i003-b-label",
      "explanation": "i003-b-explanation"
    },
    {
      "id": "i003-c",
      "label": "i003-c-label",
      "explanation": "i003-c-explanation"
    },
    {
      "id": "i003-d",
      "label": "i003-d-label",
      "explanation": "i003-d-explanation"
    }
  ],
  "correctAnswerId": "i003-a"
},
"i004": {
  "prompt": "i004-prompt",
  "answers": [
    {
      "id": "i004-a",
      "label": "i004-a-label",
      "explanation": "i004-a-explanation"
    },
    {
      "id": "i004-b",
      "label": "i004-b-label",
      "explanation": "i004-b-explanation"
    },
    {
      "id": "i004-c",
      "label": "i004-c-label",
      "explanation": "i004-c-explanation"
    },
    {
      "id": "i004-d",
      "label": "i004-d-label",
      "explanation": "i004-d-explanation"
    }
  ],
  "correctAnswerId": "i004-b"
},
"i005": {
  "prompt": "i005-prompt",
  "answers": [
    {
      "id": "i005-a",
      "label": "i005-a-label",
      "explanation": "i005-a-explanation"
    },
    {
      "id": "i005-b",
      "label": "i005-b-label",
      "explanation": "i005-b-explanation"
    },
    {
      "id": "i005-c",
      "label": "i005-c-label",
      "explanation": "i005-c-explanation"
    },
    {
      "id": "i005-d",
      "label": "i005-d-label",
      "explanation": "i005-d-explanation"
    }
  ],
  "correctAnswerId": "i005-c"
} 
```